Makefile

```diff
@@ -10,5 +10,7 @@ setup-git:
 test:
 	SNUBA_SETTINGS=test pytest -vv
 
+tests: test
```

Comment:
I added this because zsh constantly asks me if I really meant 'test' and I got tired of it.